containers/datalab/content/run.sh

```diff
@@ -109,14 +109,15 @@ if [[ -n "${INSTANCE}" ]]; then
     fi
   fi
 
-  echo "Will connect to the kernel gateway running on the GCE VM ${INSTANCE}"
+  SSH_USER=`echo ${USER_EMAIL} | cut -d '@' -f 1`
```

Comment:
> It seems USER_EMAIL could be empty if user logs in through login.ts? Do we need to set USER_EMAIL between line 67 and 68?

@qimingj Good catch; fixed.

> What is the behavior when the user is not logged in?

@chmeyers The code cannot reach to this point without the user logging in. Even if it did, it would fail during the SSH tunnel setup, since it needs the user's credentials to set up the SSH tunnel.